sentry/src/main/java/io/sentry/Span.java

```diff
@@ -317,4 +366,42 @@ public boolean isNoOp() {
   void setSpanFinishedCallback(final @Nullable SpanFinishedCallback callback) {
     this.spanFinishedCallback = callback;
   }
+
+  public void setStartDate(@NotNull SentryDate date) {
```

Comment:
`l`: should we rename it to `updateStartDate` for the sake of alignment?

Reply:
Also made it `private` since it's only accessed within `Span`